directory/directory_src.go

```diff
@@ -35,7 +36,7 @@ func (s *dirImageSource) GetManifest() ([]byte, string, error) {
 	if err != nil {
 		return nil, "", err
 	}
-	return m, "", err
+	return m, manifest.GuessMIMEType(m), err
```

Comment:
At least #145 I suppose. It feels dirty, but at least it’s now documented that way…